src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -147,13 +147,19 @@ std::pair<std::string, int> GcsClient::GetGcsServerAddress() const {
 }
 
 PythonGcsClient::PythonGcsClient(const GcsClientOptions &options) : options_(options) {
+  RAY_UNUSED(ConnectChannel());
+}
+
+Status PythonGcsClient::ConnectChannel() {
```

Comment:
I think we don't need both `ConnectChannel` and `Connect`, everything can be done in `Connect()`